using/feedback/README.md

```diff
@@ -2,8 +2,8 @@
 
 There are two types of feedback you can get on a solution:
 
-1. [Automated Feedback](./automated)
-2. [Mentor Feedback](./mentor)
+1. [Automated Feedback](./feedback/automated)
```

Comment:
Do we want to use relative links or absolute links?

Reply:
Relative in the context of the website, so this is correct.